### PR TITLE
Add README.md with screenshots

### PR DESCRIPTION
**Pull request by adrianhartanto004**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,24 @@
 # CheckPhoneSize
-Comparison check phone free size method
+Comparison check phone free size method.
+
+Device tested are in the following belows:
```
Comment (nathanielkhuana) [inline README.md]: Change to "Some of the devices that have been tested:"

**Review by nathanielkhuana — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,2 +1,24 @@
 # CheckPhoneSize
-Comparison check phone free size method
+Comparison check phone free size method.
```
Comment (nathanielkhuana) [inline README.md]: Change to "Comparison of free phone size methods"

File: README.md (hunk 1 above)
Comment (adrianhartanto004) [reply README.md]: done 👍 .

**Review by nathanielkhuana — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (adrianhartanto004) [reply README.md]: done 👍

**Pull request merged**